web/src/components/WelcomeModal.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalBody,
+    ModalFooter,
+    Button,
+  } from "@chakra-ui/react";
+  import { useState } from "react";
+  import LAPTOP_CIRCLE_ICON from "../assets/laptop-circle-icon.svg";
+  
+  export function WelcomeModal() {
+    const [openModal, setOpenModal] = useState(true);
+  
+    function handleCloseModal() {
+      setOpenModal(false);
+    }
+  
+    return (
+      <Modal isOpen={openModal} onClose={handleCloseModal} size="2xl" isCentered>
```

Comment:
Please use chakra styling rather than inline styles. see other components for examples. Docs are here: https://chakra-ui.com/